source/ballot-intro.html

```diff
@@ -10,72 +10,37 @@
+Welcome to the FHIR Release 6 (R6) ballot. 
 </p>
-<ul>
- <li><span style="background-color: #e6ffe6; border: 1px grey solid; padding-left: 3px; padding-right: 3px;"><b>Normative</b></span>: Content that is subject to normative ballot rules. Once passed, <a href="versions.html#change">strict rules</a> are imposed on future change to help ensure inter-version compatibility</li>
- <li><span style="background-color: #fff5e6; border: 1px grey solid; padding-left: 3px; padding-right: 3px;"><b>Trial Use</b></span>: Content that is undergoing <a href="versions.html#maturity">FHIR maturity Model</a> based testing. In a future ballot cycle, once sufficiently mature, it will be balloted as normative</li>
- <li><span style="background-color: #ffffe6; border: 1px grey solid; padding-left: 3px; padding-right: 3px;"><b>Informative</b></span>: Content that is advisory (e.g. implementers are not required to conform to the content), or navigational content - tables of contents, generated lists, etc.</li>
- <li><span style="background-color: #efefef; border: 1px grey solid; padding-left: 3px; padding-right: 3px;"><b>Draft</b></span>: Content added late in the balloting process that has no formal standing, but is published for visibility. It might not be suitable for use in production systems.</li>
- <li><span style="background-color: #ffcccc; border: 1px grey solid; padding-left: 3px; padding-right: 3px;"><b>Deprecated</b></span>: Content that is in the process of being removed (see <a href="versions.html#deprecation">deprecation</a>).</li>
- <li><span style="background-color: #e6ffff; border: 1px grey solid; padding-left: 3px; padding-right: 3px;"><b>External</b></span>: Content that is replicated from external standards (e.g. HL7 V2, DICOM) and is not subject to ballot comment. This content is mostly being moved out of the specification now</li>
-</ul>
-[%stu-note overall%]
-That the entirety of R5 is considered <b>trial-use</b>. Any content introduced 
-as part of R5 is subject to revision after implementation experience. If any new content is 
-labelled as normative, this should be understood to mean that the committees intention is that 
-the content will be normative in R6.
-[%end-note%]
 <p>
-  Other new content labelled trial-use may also be candidate material for normative in R6 by
-  the normal rules. Any existing content that was normative in R4 will be subject to normative 
-  change rules based on the difference from R4, not R5. R5 is a trial-use version of FHIR.
+This ballot is the first ballot of R6. This is only a draft ballot seeking review on the following items:
+
 </p>
 <p>
-There are a number of notes requesting input from reviewers as part of this ballot:
+The intention is that once this ballot is reconciled, there will be no further significant change before
+the final normative ballot. Comments are able to be made outside these specific areas as part of this ballot,

```

Comment:
I'd change this to "The intention is that there will be minimal changes to artifacts within the scope of the ballot between now and the final normative ballot, and what changes do occur will be carefully tracked and declared to balloters, which should minimize review time for these resources and pages in the final normative/STU ballot.